src/gluttony/sqs.clj

```diff
@@ -4,13 +4,13 @@
    [camel-snake-kebab.core :as csk]
    [camel-snake-kebab.extras :as cske]
    [clojure.core.async :as a]
-   [cognitect.aws.client.api.async :as aws-async]))
```

Comment:
`cognitect.aws.client.api.async` is deprcated 

https://github.com/cognitect-labs/aws-api/blob/a532fe1c14794dd8e4f9fd55750d4519ad218b8a/src/cognitect/aws/client/api/async.clj#L4-L10